ansible/playbooks/callback_plugins/custom_retry_runner.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible import constants as C
+from ansible.playbook.task_include import TaskInclude
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible.utils.color import colorize, hostcolor
+
+class CallbackModule_custom_retry_runner(CallbackModule_default):
+
+    ''''
+    This is the default callback interface, which simply prints messages
+    to stdout when new callback events are received.
+    '''
+
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'stdout'
+    CALLBACK_NAME = 'custom_retry_runner'
+
+    def __init__(self):
+
+        self._play = None
+        self._last_task_banner = None
+        super(CallbackModule, self).__init__()
+
+    def v2_runner_retry(self, result):
+        task_name = result.task_name or result._task
```

Comment:
why "or result._task", surely result.task_name should be enough?

Reply:
that's how the default ansible function is implemented, maybe it's for cases in which the name is not specified?

Reply:
Ok, I accept that